### PR TITLE
VOOD-190: Create demo project for the exhibition.

### PR DESCRIPTION
**Pull request by GinoCardillo-OST**

### CODE REVIEW

File: demos/exhibition-demo/src/playgrounds/CollectionsExample.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package playgrounds;
+
+import various.Point;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+
+public class CollectionsExample implements Playground {
```
Comment (alexandre-lagadec-ost) [inline demos/exhibition-demo/src/playgrounds/CollectionsExample.java:8]: - This is the only playground class with a `Example` suffix in its name. Is this intentional?
- Some visitors will probably ask how maps would be visualized; maybe we should add a map playground as well.

**Review by PascalSchuermannOst — APPROVED** [review]

File: demos/exhibition-demo/src/App.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import playgrounds.*;
+
+public class App {
+
+    private Playground playground = new Custom();
```
[the remark this answers is not in the document]
Comment (GinoCardillo-OST) [reply demos/exhibition-demo/src/App.java:5]: Yes, just to have a default

File: demos/exhibition-demo/src/playgrounds/CollectionsExample.java (hunk 1 above)
Comment (GinoCardillo-OST) [reply demos/exhibition-demo/src/playgrounds/CollectionsExample.java:8]: The `Example` is intentional. I can't just call it `Collections`, as it is already taken.

File: demos/exhibition-demo/src/playgrounds/Fibonacci.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package playgrounds;
+
+import java.math.BigInteger;
+
+public class Fibonacci implements Playground {
+    @Override
+    public void playground() {
+        final int fibLevel = 20;
```
[the remark this answers is not in the document]
Comment (GinoCardillo-OST) [reply demos/exhibition-demo/src/playgrounds/Fibonacci.java:8]: It takes ~13 milliseconds. You must be confusing it with something else.

**Pull request merged**